src/util-internal.h

```diff
@@ -102,7 +102,7 @@ _netplan_state_get_vf_count_for_def(const NetplanState* np_state, const NetplanN
 NETPLAN_INTERNAL gboolean
 _netplan_netdef_get_sriov_vlan_filter(const NetplanNetDefinition* netdef);
 
-NETPLAN_INTERNAL gboolean
+NETPLAN_INTERNAL NetplanCriticalOption
 _netplan_netdef_get_critical(const NetplanNetDefinition* netdef);
```

Comment:
Note: This is `NETPLAN_INTERNAL` API, as also desribed by the `_netplan` prefix. It's supposed to be used by Netplan tools/CLI only, not by external consumers. Although, changing the signature is an API breaking change, I think it's fine as we're the only ones affected.